entity-service/src/main/kotlin/com/egm/stellio/entity/service/IAMListener.kt

```diff
@@ -23,56 +23,56 @@ class IAMListener(
 
     @StreamListener("cim.iam")
     fun processMessage(content: String) {
-        val entityEvent: EntityEvent = parseEntityEvent(content)
+        val authorizationEvent: AuthorizationEvent = parseAuthorizationEvent(content)
```

Comment:
AFAIK, the payload of an authorization event conforms to an entity event payload. So we should not have a special model for authorization events.

Reply:
The payload of an authorization event conforms to the old entity event payload.
For this reason, i re-used the old model (refactored to AuthorizationEvent model).